api/src/main/kotlin/nl/avisi/anowalke/controllers/RestController.kt

```diff
@@ -5,34 +5,31 @@ import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import mu.KotlinLogging
+import nl.avisi.anowalke.TranslateConnector
 import nl.avisi.anowalke.dto.ExpressionsDto
 import nl.avisi.anowalke.services.FilterService
-import nl.avisi.anowalke.services.VertaalService
+import nl.avisi.anowalke.services.translation.MicrosoftVertaalService
 
 @RestController
 @RequestMapping("/")
 class RestController(
-    private val vertaalService: VertaalService,
+    private val vertaalService: TranslateConnector,
```

Comment:
Bijna wat ik verwacht had. Enige is dat ik als je het zo doet hem ook TranslateService zou noemen en niet Connector. Connector zou naar mijn idee logischer zijn als het weer iets is wat je in een service aanroept.